e2e/config/e2e_load_test.config.tmpl.yaml

```diff
@@ -28,28 +28,48 @@ e2e_load_test_config:
   # Load Test Mode
   # Tests run against a specified gateway URL (local or public)
   # Examples:
-  #   - Local:  "http://localhost:3091/v1"
   #   - Public: "https://rpc.grove.city/v1"
+  #   - Local:  "http://localhost:3091/v1"
   load_test_config:
     # [Required] Custom PATH gateway URL
-    gateway_url_override: ""
+    #
+    # The default field uses the Grove Portal as the gateway URL.
+    #
+    # If you are wishing to test against a local PATH instance,
+    # you can override this field with the local PATH gateway URL 
+    # (default: "http://localhost:3069/v1")
+    gateway_url_override: "https://rpc.grove.city/v1"
+
+    # [Optional] Insert service ID into subdomain per test case
+    #     Example: "F00C.rpc.grove.city/v1"
+    #
+    # TODO_TECHDEBT(@commoddity): Remove after header-based service selection is supported
+    #     See: https://github.com/buildwithgrove/infrastructure/issues/91
+    use_service_subdomain: true
+
     # [Required] Custom user identifier for the test (i.e. portal-application-id)
+    #
+    # DEV_NOTE: If you do not have a Grove Portal Application ID, get one by visiting:
+    #     https://www.portal.grove.city
+    #
+    # This field is NOT required if you are testing against a local PATH instance.
     portal_application_id: ""
-    # [Optional] Insert service ID into subdomain per test case
-    # - Example: "F00C.rpc.grove.city/v1"
-    # - TODO_TECHDEBT(@commoddity): Remove after header-based service selection is supported
-    # - See: https://github.com/buildwithgrove/infrastructure/issues/91
-    use_service_subdomain: false
+
     # [Optional] Custom API key for the test (i.e. portal-api-key)
+    #
+    # DEV_NOTE: This field is required if your Grove Portal Application ID requires 
```

Comment:
Should we change this to: "Not required for local testing, needed for all other test cases"